subsys/bluetooth/mesh/Kconfig

```diff
@@ -51,6 +51,68 @@ endif # BT_MESH_RELAY
 
 endif
 
+config BT_MESH_NLC_PERF_SUPPORT
+	bool "Enable NLC performance configurations"
+	help
+	  Enables performance configurations required for NLC profiles.
```

Comment:
```suggestion
	  Enables performance configurations required for NLC profiles to be qualifiable.
```
?

Reply:
Should we wait to mention this until they are actually possible to qualify?

Reply:
Right, good point